homeassistant/components/switch/flux.py

```diff
@@ -151,8 +151,9 @@ def turn_off(self, **kwargs):
         self.update_ha_state()
 
     # pylint: disable=too-many-locals
-    def flux_update(self, now=dt_now()):
+    def flux_update(self, service=None):
```

Comment:
This parameter name is incorrect. You sometimes pass in nothing, sometimes a time.

This would be more correct:

```python
def flux_update(self, now=None):
    if now is None:
        now = dt_now()
```